corehq/apps/sms/views.py

```diff
@@ -707,7 +707,7 @@ def get_contact_info(domain):
     case_ids = []
     mobile_worker_ids = []
     data = []
-    for p in PhoneNumber.by_domain(domain):
+    for p in PhoneNumber.by_domain(domain).filter(is_two_way=True):
```

Comment:
this was actually wrong before - here we're producing a list of contacts that we can use the chat functionality with, which we should only get from the set of two way numbers